doc/getting-started/getting-started.rst

```diff
@@ -170,10 +181,10 @@ To set up the ACRN build environment on the development computer:
       cd ~/acrn-work
       git clone https://github.com/projectacrn/acrn-hypervisor.git
       cd acrn-hypervisor
-      git checkout v2.7
+      git checkout master # TODO determine what to check out
```

Comment:
Eventually this will checkout the v3.0 branch

Reply:
We could use latest tag "acrn-2022w18.4-180000p" for now.

Reply:
Done